proc_Hahn_echoph.py

```diff
@@ -7,13 +7,13 @@
 
 
 signal_pathway = {"ph1": 1, "ph2": 0}
-with psd.fl_mod() as fl:
```

Comment:
`fl_mod` is actually defined and belongs to proc scripts -- search for `class fl_mod` --> it supplies other plotting styles.

Reply:
thank you! was wondering why I got the issue with fl_mod not belonging to pyspecdata